app/lib/test_applications.rb

```diff
@@ -174,6 +174,9 @@ def put_application_choice_in_state(choice, state)
     when :offer_withdrawn
       make_offer(choice)
       withdraw_offer(choice)
+    when :offer_deferred
+      make_offer(choice)
```

Comment:
I think this breaks the rules of the state machine — should we `accept_offer` first? And maybe we should sometimes have `recruited` applications.

Reply:
Well spotted. We already have `recruited` applications in the test data (showing up as 'Conditions met')

Reply:
Ah, you mean deferred applications _via_ `recruited`. Will add.